src/components/pill-selector-dropdown/mapInputValuesToOptionsParser.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import uniqBy from 'lodash/uniqBy';
+import escapeRegExp from 'lodash/escapeRegExp';
+
+import parseCSV from '../../utils/parseCSV';
+
+import type { Option } from './flowTypes';
+
+// Custom input parser for the PillSelectorDropdownField that automatically maps
+// a list of comma separated values to their respective Option objects. The parser
+// will try to find exact but case-insenstive matches using the option's displayText
+// property. If a token does not match displayText, the parser will also try to match
+// the string representation of the value property. This last match is useful for country
+// codes and other similar use cases.
+function mapInputValuesToOptionsParser(
+    inputValue: string,
+    options: Array<Option>,
+    selectedOptions: Array<Option>,
+): Array<Option> {
+    let mappedOptions;
+    const inputTokens = parseCSV(inputValue);
```

Comment:
Would we need to use the custom parseItems() prop here? Like in PillSelectorDropdown.js parsePills()?
let pills = parseItems ? parseItems(inputValue) : parseCSV(inputValue);

Reply:
Nope, this function is actually meant to be used as the `parseItems` prop itself.